### ISSUE TITLE
USWDS - Bug: Date picker year button overlapping with days below

### ISSUE THREAD

**Issue by smustgrave** [uswds/uswds#6069]
### Describe the bug

When open the date picker year button appears to wrap and overlaps with days below

### Steps to reproduce the bug

1. Have a 2 grid layout 3-9 with grid-gap-4
2. Add a date picker in the left column
3. Open date picker

### Expected Behavior

Date picker remains inline no matter what

Workaround `usa-date-picker__calendar__cell usa-date-picker__calendar__month-labe` added `display: flex`

### Related code

<div class="padding-2">
  <div class="grid-row">
    <div class="grid-col-3">
      <div class="usa-form-group">
        <label class="usa-label" id="appointment-date-label" for="appointment-date"
          >Appointment date</label
        >
        <div class="usa-hint" id="appointment-date-hint">mm/dd/yyyy</div>
        <div class="usa-date-picker">
          <input
            class="usa-input"
            id="appointment-date"
            name="appointment-date"
            aria-labelledby="appointment-date-label"
            aria-describedby="appointment-date-hint"
          />
        </div>
      </div>
    </div>
    <div class="grid-col-9">.grid-col-9</div>
  </div>

### Screenshots

![image](https://github.com/user-attachments/assets/3c18d56c-1b1a-4dc2-b85b-a64a14206c55)

### System setup

- USWDS version 3.8.1
- Browser Chrome

### Additional context

See slack thread https://gsa-tts.slack.com/archives/C3F14AHSQ/p1726526329713889

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.

**Comment by mlloydbixal** [comment]
@dct2023 there was a failing unit test that has been resolved. This is good to go for peer review!

**Comment by hallekfields** [comment]
Hi! @mandylloyd is there any movement on this issue?
